src/dcqc/suites/suite_abc.py

```diff
@@ -171,3 +232,37 @@ def to_dict(self) -> SerializedObject:
             "tests": test_dicts,
         }
         return suite_dict
+
+    @classmethod
+    def from_dict(cls, dictionary: SerializedObject) -> SuiteABC:
```

Comment:
Nit: These `from_dict` functions would fail with a `KeyError` error if a dictionary with missing keys was passed in.  Not that this is a huge deal, but just some food for thought.  Should we be concerned about those scenarios?

Reply:
A `KeyError` would be appropriate in that situation. In theory, these dictionaries should only be generated by dcqc's `to_dict()` methods. We should also have tests that ensure that we can perform a serialize/deserialize round-trip without any issues. For now, I don't think we need to support deserialization of arbitrary JSON files.